src/terra-dev-site/app/Application.1.app.mdx

```diff
@@ -0,0 +1,22 @@
+# terra-application
+
+A React-based application platform from Cerner.
+
+## Why create an application platform?
+
+As we have developed web applications at cerner we have found that there are many questions that must be answered at the outset of an application's development and that many of those question have a common answer that can be provided by an application platform.
+
+- How will this application consume existing components developed for other applications? How will interoperability be defined? How will dependencies be managed?
+- How will this application have a user experience that is consistent with that of other applications? How will that consistency be maintained over time?
+
+The terra-application framework seeks to answer these questions and simplify the process of maintaining web applications on the Orion platform.
+
+### Component Composition
+
+The framework provides a large number of features using React's Context APIs. An application built with the framework is guaranteed to provide those framework features to the components it renders, and reusable components can be developed with the assumption that the framework will provide them. Any components requiring the same major version of terra-application will be interoperable.
+
+The terra-application package brings all the required logic into a single location, resulting in simplified dependency management across the suite of developed components.
+
+### UX Consistency
+
+The framework utilizes the full suite of Terra's components to ensure visual consistency and accessibility for applications. The framework also provides simplified APIs for common design patterns like loading overlays, modals, and error view presentation to further promote consistency.
```

Comment:
Can we add a getting started & peer dep section like we do with the components? 

It might be nice to provide an Overview section to outline the components & contexts that are provided. The terra-ui deployment goes straight to references, which feel like you need to read it all to know where to start. I think the references are nice if you know what you are looking for

Reply:
IMO the peer dep section adds no value and unless we do work is all but guaranteed to fall out of date. So I don't want to add it.

Reply:
....I do agree with the peer dep part.

Reply:
Adding the overview section makes sense. I'll work on that.